linux_os/guide/system/software/system-tools/package_krb5-workstation_removed/rule.yml

```diff
@@ -21,6 +21,9 @@ identifiers:
     cce@rhel7: 82930-9
     cce@rhel8: 82931-7
 
+references:
+    srg: SRG-OS-000095-GPOS-00049
```

Comment:
The krb5-workstation package is not installed by default, however the rule is in the OSPP profile.  Can the rule be removed there and then unmapped from the STIG?  I think doing that will allow the OSPP profile and the STIG to stay closer in sync.

Reply:
@stevegrubb can you help with clarification why `krb5-workstation` package is being removed by OSPP profile?

Reply:
It is removed because NIAP does not list krb5 as an approved authentication method. NIST also does not allow its key derivation function. So, to reduce attack surface and take away things that are forbidden, we remove it.

Reply:
Nitpick about the mapping. Agreement the rule should be there per @stevegrubb 's comments. The current SRG mapping is about disabling non-essential capabilities:

SRG-OS-000095-GPOS-00049 |   | The operating system must be configured to disable non-essential capabilities.
-- | -- | --

Because default installations do not have this package, not sure if this aligns well to disabling non-essential things.

What about swapping it for ``SRG-OS-000120-GPOS-00061`` ? That control reads:

SRG-OS-000120-GPOS-00061 |   | The operating system must use mechanisms meeting the requirements of applicable federal laws, Executive orders, directives, policies, regulations, standards, and guidance for authentication to a cryptographic module.
-- | -- | --

Reply:
added SRG-OS-000120-GPOS-00061